cmake/pcl_targets.cmake

```diff
@@ -101,28 +101,28 @@ macro(PCL_SUBSYS_DEPEND _var _name)
             if(NOT _status)
                 set(${_var} FALSE)
                 PCL_SET_SUBSYS_STATUS(${_name} FALSE "Requires ${_dep}.")
-            else(NOT _status)
+            else()
                 PCL_GET_SUBSYS_INCLUDE_DIR(_include_dir ${_dep})
                 include_directories(${PROJECT_SOURCE_DIR}/${_include_dir}/include)
-            endif(NOT _status)
+            endif()
         endforeach(_dep)
-        endif(SUBSYS_DEPS)
+        endif()
         if(SUBSYS_EXT_DEPS)
         foreach(_dep ${SUBSYS_EXT_DEPS})
             string(TOUPPER "${_dep}_found" EXT_DEP_FOUND)
             if(NOT ${EXT_DEP_FOUND} AND (NOT (${EXT_DEP_FOUND} STREQUAL "TRUE")))
                 set(${_var} FALSE)
                 PCL_SET_SUBSYS_STATUS(${_name} FALSE "Requires external library ${_dep}.")
-            endif(NOT ${EXT_DEP_FOUND} AND (NOT (${EXT_DEP_FOUND} STREQUAL "TRUE")))
+            endif()
         endforeach(_dep)
```

Comment:
You can apply the same fix for `foreach` loops.

Reply:
Done